packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -279,18 +301,30 @@ export class KlevuQuicksearch {
       if (this.usePersonalisation === undefined && settings?.klevu_uc_userOptions.enablePersonalisationInSearch) {
         this.usePersonalisation = true
       }
-    }
-    const showPopularProducts = settings?.klevu_uc_userOptions?.noResultsOptions.showPopularProducts
-    if (showPopularProducts) {
-      const trendingProductsQuery = await KlevuFetch(
-        trendingProducts({
-          limit: this.simpleResultCount,
-        })
-      )
-      const resultObject = trendingProductsQuery.queriesById("trendingProducts")
-      if (resultObject) {
-        this.trendingProducts = resultObject.records
-        this.#emitChangedData()
+      if (this.showPopularKeywords === undefined)
+        this.showPopularKeywords = settings?.klevu_uc_userOptions.noResultsOptions.showPopularKeywords
+      if (this.showTrendingProducts === undefined)
+        this.showTrendingProducts = settings?.klevu_uc_userOptions.noResultsOptions.showPopularProducts
+      if (this.showRecentlyViewedProducts === undefined)
+        this.showRecentlyViewedProducts = settings?.klevu_uc_userOptions.showRecentlyViewedItems
+      if (this.lastClickedProductsCaption === undefined)
+        this.lastClickedProductsCaption = settings?.klevu_uc_userOptions.showRecentlyViewedItemsCaption || ""
+      if (this.trendingCaption === undefined)
+        this.trendingCaption = settings?.klevu_uc_userOptions.showTrendingProductsCaption || ""
```

Comment:
Instead of empty strings here as fallback we should use the translation like `getTranslation("quicksearch.tTrendingCaption")`